### DISCUSSION TITLE
Feature request: system tray icon with ability to perform functions

### DISCUSSION THREAD

**Discussion by geogolem** [Taxel/PlexTraktSync#749] [Ideas]
### Feature description
It would be cool if the app could run in the system tray with the ability to enable/disable the watch, schedule syncs at specific intervals and/or manually kick off syncs

**Comment by glensc** [comment]
please fill feature request in the title so the issue listing is not a clickbait

**Comment by glensc** [comment]
perhaps this is out of scope of this project. as the project is now present in pypi, perhaps the GUI can be a standalone project.

- https://pypi.org/project/PlexTraktSync/

**Comment by glensc** [comment]
python in browser could be useful here:
- https://urllib3.readthedocs.io/en/stable/reference/contrib/emscripten.html
- https://pyodide.org/en/stable/usage/quickstart.html

**Comment by glensc** [comment]
also maybe:
- https://www.pythonguis.com/tutorials/packaging-pyqt5-apps-fbs/
